### PR TITLE
about us hero image

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: includes/data/post.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[
```
Comment (ElXyfer) [inline includes/data/post.json]: this code is not part of the hero image.

File: theme/sass/components/_stats-bar.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.stats-bar {
```
Comment (ElXyfer) [inline theme/sass/components/_stats-bar.scss]: this code is not part of the hero image.

File: includes/data/post.json (hunk 1 above)
Comment (katlazarova) [reply includes/data/post.json]: done

File: theme/sass/components/_stats-bar.scss (hunk 2 above)
Comment (katlazarova) [reply theme/sass/components/_stats-bar.scss]: done

File: theme/sass/components/_hero-image.scss [hunk 3]
```diff
@@ -11,12 +11,24 @@
...
+        padding: 0 2rem;
+        border-radius: 10px;
+        background-color: transparentize($secondary-color, 0.7);
+
+        @include breakpoint($md) {
+            max-width: 200rem;
+            position: absolute;
+            left: 10vw;
+            bottom: 5vw;
+            font-size: 30px;
```
Comment (ElXyfer) [inline theme/sass/components/_hero-image.scss]: Remove font size from here and add it to the specific element you wanna make bigger, also 30px was too large.

File: theme/sass/components/_hero-image.scss [hunk 4]
```diff
@@ -11,12 +11,24 @@
...
         text-align: left;
         font-weight: bold;
         color: white;
+        padding: 0 2rem;
+        border-radius: 10px;
+        background-color: transparentize($secondary-color, 0.7);
+
+        @include breakpoint($md) {
+            max-width: 200rem;
+            position: absolute;
```
Comment (ElXyfer) [inline theme/sass/components/_hero-image.scss]: Remove position: absolute from here its not doing anything.

File: theme/sass/components/_hero-image.scss [hunk 5]
```diff
@@ -11,12 +11,24 @@
...
-        bottom: 10vw;
+        left: 5vw;
+        bottom: 5vw;
+        max-width: 15rem;
         text-align: left;
         font-weight: bold;
         color: white;
+        padding: 0 2rem;
+        border-radius: 10px;
+        background-color: transparentize($secondary-color, 0.7);
```
Comment (ElXyfer) [inline theme/sass/components/_hero-image.scss]: Make this less transparent, reduce by maybe a 0.3 or 0.2

File: theme/sass/components/_hero-image.scss (hunk 3 above)
Comment (katlazarova) [reply theme/sass/components/_hero-image.scss]: done

File: theme/sass/components/_hero-image.scss (hunk 4 above)
Comment (katlazarova) [reply theme/sass/components/_hero-image.scss]: done

File: theme/sass/components/_hero-image.scss (hunk 5 above)
Comment (katlazarova) [reply theme/sass/components/_hero-image.scss]: done

**Pull request merged**